.readthedocs.yaml

```diff
@@ -14,3 +14,8 @@ sphinx:
 python:
   install:
     - requirements: doc/requirements.txt
+
+search:
+  ignore:
+    - "doxygen-output/*"
+    - "_static/*"
```

Comment:
Wonder if the default list should be kept here? Second example of this section:

https://docs.readthedocs.io/en/stable/config-file/v2.html#search-ignore

```
     # Defaults
     - search.html
     - search/index.html
     - 404.html
     - 404/index.html'
```

Otherwise, lgtm and :crossed_fingers:

Reply:
Thanks for checking, adjusted.